code/modules/hydroponics/grown/apple.dm

```diff
@@ -31,8 +31,8 @@
 /obj/item/seeds/apple/poisoned
 	product = /obj/item/reagent_containers/food/snacks/grown/apple/poisoned
 	mutatelist = list()
-	reagents_add = list("zombiepowder" = 0.5, "vitamin" = 0.04, "nutriment" = 0.1)
-	rarity = 50 // Source of cyanide, and hard (almost impossible) to obtain normally.
+	reagents_add = list("zombiepowder" = 0.1, "vitamin" = 0.04, "nutriment" = 0.1)
+	rarity = 50 // Source of zombie powder, and hard (almost impossible) to obtain normally.
```

Comment:
i like how you corrected the comment but still didn't change the fact that it isn't almost impossible to obtain normally, it's actually kinda easy now